google/cloud/bigquery/client.py

```diff
@@ -2833,9 +2833,6 @@ def load_table_from_json(
 
         new_job_config.source_format = job.SourceFormat.NEWLINE_DELIMITED_JSON
 
-        if new_job_config.schema is None:
```

Comment:
We are still getting test failures with this change:

```

>           raise exceptions.from_http_response(exc.response)
E           google.api_core.exceptions.BadRequest: 400 POST https://bigquery.googleapis.com/upload/bigquery/v2/projects/precise-truck-742/jobs?uploadType=multipart: No schema specified on job or table.
```

I think we can only omit these if the table already exists and we are appending rows.

Reply:
Thanks, I think I made a mistake in my test code and created the table with schema in it.